addons/event_crm/views/crm_lead_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="action_view_lead_registrations" model="ir.actions.act_window">
```

Comment:
Guidelines ! Here it should be something like event_registration_action_from_lead

Reply:
By the way if event_views is imported before you can put the action in event_views.xml